pkg/daemon/agent/flexvolume/server_test.go

```diff
@@ -32,9 +32,11 @@ func TestConfigureFlexVolume(t *testing.T) {
 	driverFile := path.Join(driverDir, flexvolumeDriverFileName)
 	os.OpenFile(driverFile, os.O_RDONLY|os.O_CREATE, 0755)
 
-	err := configureFlexVolume(driverFile, driverDir)
+	driverName := "rook"
+	os.Setenv("POD_NAMESPACE", driverName)
+	defer os.Setenv("POD_NAMESPACE", "")
```

Comment:
should this be set back to the original value of the env var instead of to blank?

Reply:
in the unit tests I would expect the original value to be blank